Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue5132.cs

```diff
@@ -0,0 +1,76 @@
+﻿using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+using Xamarin.Forms.Controls;
+
+#if UITEST
+using Xamarin.Forms.Core.UITests;
+using Xamarin.UITest;
+using NUnit.Framework;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+#if UITEST
+	[Category(UITestCategories.Shell)]
+#endif
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Github, 5132, "Unable to specify automation properties on the hamburger/flyout icon", PlatformAffected.Default)]
+	public class Issue5132 : TestShell
+	{
+		string _idIconElement = "shellIcon";
+		string _titleElement = "Connect";
+		protected override void Init()
+		{
+			Title = "Shell";
+			FlyoutIcon = new FontImageSource
+			{
+				Glyph = "\uf2fb",
+				FontFamily = DefaultFontFamily(),
+				Size = 20,
+				AutomationId = _idIconElement
+			};
+			FlyoutIcon.SetValue(AutomationProperties.HelpTextProperty, "This as Shell FlyoutIcon");
+			FlyoutIcon.SetValue(AutomationProperties.NameProperty, "SHELLMAINFLYOUTICON");
+			Items.Add(new FlyoutItem
+			{
+				Title = _titleElement,
+				Items = {
+					new Tab { Title = "library",
+						Items = {
+									new ContentPage { Title = "Library",  Content = new Label  { Text = "Turn accessibility on and make sure the help text is read" } }
```

Comment:
Can we nest the content in a scrollview for now? Because of that bug on iOS where content can't be seen?